docs/api/rest_api/rest_api_reference/input/examples/content/objects/content_id/versions/version_no/PATCH/VersionUpdate.xml.example

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<VersionUpdate xmlns:p="http://ez.no/API/Values"
+<VersionUpdate xmlns:p="http://ibexa.co/API/Values"
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://ez.no/API/Values ../VersionUpdate.xsd ">
+  xsi:schemaLocation="http://ibexa.co/API/Values ../VersionUpdate.xsd ">
```

Comment:
I'm surprised by this syntax. I didn't find an example to validate that.
([ContentTest doesn't have any schema attributes.](https://github.com/ibexa/rest/blob/v4.6.7/tests/bundle/Functional/ContentTest.php#L296-L304))